utils/test-utils/README.md

```diff
@@ -0,0 +1,254 @@
+# Testing-kotlin
+
+In this module you can find the different utilities that we have created in order to help us to implement UI tests on our libraries.
+
+These utilities were originally created as a way to reuse code on our different UI tests without the need to copy-paste useful function on each library module.
+Since we think that there is value on these utilities to be shared with the developer community we have created a testing component that can be added as a external dependency to any project that want to implement UI tests targeting dual-screen and foldable devices.
+
+## Add to your project
+
+1. Make sure you have **mavenCentral()** repository in your top level **build.gradle** file:
+
+    ```gradle
+    allprojects {
+        repositories {
+            google()
+            mavenCentral()
+         }
+    }
+    ```
+
+2. Add dependencies to the module-level **build.gradle** file (current version may be different from what's shown here):
+
+    ```gradle
+    implementation "com.microsoft.device.dualscreen.testing:testing-kotlin:1.0.0-alpha1"
```

Comment:
Should we also update this version to 1.0.0-alpha2?

Reply:
totally! good catch!

Reply:
Fixed